zeek/README.md

```diff
@@ -0,0 +1,201 @@
+## Overview
+
+[Zeek][6] is the world's leading platform for network security monitoring. It interprets what it sees and creates compact, high-fidelity transaction logs, and file content. It can create fully customized output, suitable for manual review on disk or in a more analyst-friendly tool like a security and information event management (SIEM) system.
+
+This integration provides visualization and log enrichments for Connection Logs, DNS & DHCP Logs, Network Protocols, Files, Detections, and Miscellaneous event types.
+
```

Comment:
Please clarify or change "log enrichments". Also, would like to have added:
- what are some examples of logs?
- why would users want to ingest them into the platform?
- what benefits do they get from provided visualizations?

Reply:
Done